src/nunit.analyzers.tests/NullConstraintUsage/NullConstraintUsageAnalyzerTests.cs

```diff
@@ -33,6 +33,16 @@ public void AnalyzeWhenActualIsNonNullableValueTypeDelegate(string constraint)
             AnalyzerAssert.Diagnostics(analyzer, expectedDiagnostic, testCode);
         }
 
+        [TestCase("Is.Null")]
+        [TestCase("Is.Not.Null")]
+        public void ValidWhenActualIsTaskDelegate(string constraint)
```

Comment:
```suggestion
        public void AnalyzWhenActualIsTaskDelegate(string constraint)
```

Reply:
Renamed to AnalyzeWhenActualIsTaskDelegate (added e to your suggestion)